addons/website_blog/views/website_blog_templates.xml

```diff
@@ -112,7 +112,7 @@ list of filtered posts (by date or tag).
     Replace top-banner content with the latest published post
 -->
 <template id="opt_blog_cover_post" name="Top banner - Name / Latest Post" inherit_id="website_blog.blog_post_short" active="True" customize_show="True">
-    <xpath expr="//div[@id='o_wblog_blog_top_droppable']" position="replace">
+    <xpath expr="//div[@id='o_wblog_blog_top']" position="inside">
```

Comment:
I actually don't think this is the right fix: this is a change in behavior, no? We have a task in master reviewing that behavior anyway, let's not change it as a FIX.

In fact I think there is very simpler fix: juste increase the sequence of this view. That way the xpath will apply after oe_structure customizations. Maybe in master we could also create oe_structure customization with a smaller sequence so that we do not have that kind of problem anymore. Anyways: in stable, let's increase the sequence. You can actually make a very simple python patch in the Customize feature so that it works for people without -u as well.

Reply:
We don't have a priority... we have worse: a priority named "sequence"
```python
priority = fields.Integer(string='Sequence', default=16, required=True)
```
:')